docs/support/tutorials/ml-llm.md

```diff
@@ -180,33 +180,101 @@ Face](https://huggingface.co/blog/mlabonne/sft-llama3), which also
 covers the [Unsloth library](https://github.com/unslothai/unsloth).
 
 
-## Using quantization
+## Quantization
 
-Using the `bitsandbytes` library, you can also use 4-bit
-quantization. [Quantization has been integrated into Hugging Face
-Transformers as
-well](https://huggingface.co/blog/4bit-transformers-bitsandbytes).
+Quantization is a process that converts the weights and activations within an LLM from high-precision values, such as 32-bit floating-point, to lower-precision ones, such as an 8-bit integer. This leads to a significant decrease in overall model size, leading to smaller memory needs with a slight drop in accuracy. You can learn more about quantization for example in [this tutorial](https://www.datacamp.com/tutorial/quantization-for-large-language-models).
 
+Quantization can be done during inference or training phase. [Post-Training Quantization (PTQ)](https://docs.pytorch.org/TensorRT/ts/ptq.html) involves quantizing a pre-trained model during the inference phase. [Quantization-Aware Training (QAT)](https://www.ibm.com/think/topics/quantization-aware-training) is applied during training to simulate the effects of quantization, resulting in a model more robust to quantization noise. This guide focuses on Post-Training Quantization.
+
+It can take hours to quantize very large models, but luckily many models already have a quantized version available, for example in Hugging Face. You can look for quantized models by a suffix in the model name indicating a quantization method, such as AWQ, GPTQ, or GGUF, or alternatively, model precision, such as 8bit or 4bit. 
```

Comment:
AWQ, GPTQ, GGUF. These could maybe be linked as they are not so obvious? I guess some of them are explained below.